samples/built-in-policy/deploy-logAnalytics-diagnostic-setting-for-datalakeanalytics/azurepolicy.rules.json

```diff
@@ -0,0 +1,16 @@
+{
+  "if": {
+    "field": "type",
+    "equals": "microsoft.datalakeanalytics/accounts"
+  },
+  "then": {
+    "effect": "[parameters('effect')]",
+    "details": {
+      "type": "Microsoft.Insights/diagnosticSettings",
+      "existenceCondition": {
+        "field": "Microsoft.Insights/diagnosticSettings/logs.enabled",
+        "equals": "true"
+      }
+    }
+  }
+}
```

Comment:
Please include the full policy rule  - including the deployment section - in this file (everything inside the "policyRule" section in the azurepolicy.json file)